app/profile-page.html

```diff
@@ -0,0 +1,143 @@
+                     <label class="sr-only" for="loginEmail">Email</label>
+                     <input type="email" class="form-control" id="loginEmail" placeholder="Email" ng-model="usr_email" >
+                  </div>
+                  <p class="text-danger" ng-if="error.show"><small>{{error.message}}</small></p>
+                  <fieldset>
+                     <ng-form>
+                        <div class="form-group">
+                           <label class="sr-only" for="loginPassword">Current Password</label>
+                           <input type="password" ng-pattern="/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/" class="form-control" id="loginPassword" placeholder="Current Password" ng-model="usr_password" >
+                           <!--<div ng-show="form.$submitted || form.pass.$touched">
+                              <span ng-show="form.pass.$error.required">Enter your current password.</span>
+                              <span ng-show="form.pass.$error.pattern">Password must have 8 character, 1 letter and 1 number.</span>
+                           </div> -->
+                        </div>
+                        <div class="form-group">
+                           <label class="sr-only" for="newPassword">New Password</label>
+                           <input type="password" ng-pattern="/^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/" class="form-control" id="newPassword" placeholder="New Password" ng-model="usr_newPassword" >
+                           <!-- <div ng-show="form.$submitted || form.pass.$touched">
+                              <span ng-show="form.pass.$error.required">Enter your new password.</span>
+                              <span ng-show="form.pass.$error.pattern">Password must have 8 character, 1 letter and 1 number.</span>
+                           </div> -->
+                        </div>
+                        <div class="form-group">
+                           <label class="sr-only" for="confirmPassword">Confirm Password</label>
+                           <input type="password" class="form-control" id="confirmPassword" placeholder="Confirm Password" ng-model="usr_confirmPassword" >
+                          <!-- <div ng-show="form.$submitted || (form.passCon.$touched && form.pass.$touched)">
+                              <div ng-show="form.passCon.$error.compareTo">Passwords must match.</div>
+                           </div> -->
+                        </div>
+                  </fieldset>
+				  <br>
+                  <div class="form-group">
+                  <button type="submit" class="btn btn-success pull-right">Confirm</button>
+                  </div>
+                  </ng-form>
+				  <br>
+                  <div class="form-group">
+                     <!--<button type="submit" class="btn btn-success pull-right">Save Changes</button> -->
+					 <div id="uploadBtn" class="btn btn-success pull-right" ng-click="getFile()">Save Changes</div>
```

Comment:
Optional - Tempted to say to remove all method calls, like `getFile()`, in this Pull request. Just so whoever works on this next doesn't get confused or miss one?